### PR TITLE
Feature #223 - Login E2E test

### LINKED ISSUE #233: Login DOM test
This test should check the next scenario:
- The user creates an account
- The user closes the browser (you can simulate this by just calling clearPersona on background script).
- The user launches the app again, it should be redirected to login route
- After introducing the password user is redirected to account status view.

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will add E2E test for Login route to check extension behaviour in case of closed browser. It will close #233.

### CODE REVIEW

File: packages/sanes-chrome-extension/src/routes/login/index.e2e.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          (window as IovWindowExtension).clearPersona();
+        },
+      );
+
+      //Simulating reload
+      await page.goto(`chrome-extension://${EXTENSION_ID}/index.html`, {
+        waitUntil: 'networkidle2',
+      });
+      await findRenderedE2EComponentWithId(page, LOGIN_ROUTE);
+
```
Comment (apanizo) [inline packages/sanes-chrome-extension/src/routes/login/index.e2e.spec.ts]: I am just missing here simulate user introducing password and checking is redirected to account status view.

If you take a look into the login DOM test, we do not check if this functionality works, and having this awesome test I think is cool to do it.

File: packages/sanes-chrome-extension/src/extension/background/model/backgroundscript/index.ts [hunk 2]
```diff
@@ -86,13 +87,14 @@ class Backgroundscript {
...
   }
 
-  public registerActionsInBackground(): void {
-    (window as IovWindowExtension).getQueuedRequests = () => this.signingServer.getPendingRequests();
-    (window as IovWindowExtension).createPersona = (pss, mn) => this.createPersona(pss, mn);
-    (window as IovWindowExtension).loadPersona = pss => this.loadPersona(pss);
-    (window as IovWindowExtension).createAccount = () => this.createAccount();
-    (window as IovWindowExtension).getPersonaData = () => this.getPersonaData();
-    (window as IovWindowExtension).hasStoredPersona = () => this.hasStoredPersona();
+  public registerActionsInBackground(windowExtension: IovWindowExtension): void {
```
Comment (apanizo) [inline packages/sanes-chrome-extension/src/extension/background/model/backgroundscript/index.ts]: Good job.

File: packages/sanes-chrome-extension/src/extension/backgroundscript.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
 /*global chrome*/
 import { generateErrorResponse } from './background/errorResponseGenerator';
-import Backgroundscript from './background/model/backgroundscript';
+import Backgroundscript, { IovWindowExtension } from './background/model/backgroundscript';
 import { updateExtensionBadge } from './background/updaters/extensionBadgeUpdater';
 
 const backgroundScript = new Backgroundscript();
-backgroundScript.registerActionsInBackground();
+backgroundScript.registerActionsInBackground(window as IovWindowExtension);
```
Comment (apanizo) [inline packages/sanes-chrome-extension/src/extension/backgroundscript.ts]: 👌

File: packages/sanes-chrome-extension/src/utils/test/e2e.ts [hunk 4]
```diff
@@ -27,6 +29,20 @@ export async function createPage(browser: Browser): Promise<Page> {
   return page;
 }
 
+export async function getBackgroundPage(browser: Browser): Promise<Page> {
```
Comment (apanizo) [inline packages/sanes-chrome-extension/src/utils/test/e2e.ts]: Remind me when we do the retrospective feedback once the extension is done, to bring the topic for creating another project sharing all test utilities avoiding duplicated code.

Good job by the way.

Comment (albertandrejev) [reply packages/sanes-chrome-extension/src/utils/test/e2e.ts]: eeh, I want too not to forget about it :)

**Pull request merged**